### PR TITLE
fix: how to patch changes to user?

### PR DESCRIPTION
**Pull request by danielgent**

### CODE REVIEW

File: src/routes/tsoa/users.ts [hunk 1]
```diff
@@ -129,7 +129,10 @@ export class UsersController extends Controller {
       await userInstance.setPassword.mutate(body.password);
     }
 
-    return userInstance.getProfile.query();
+    return userInstance.patchProfile.mutate({
+      ...body,
+      tenantId,
+    });
```
[the remark this answers is not in the document]
Comment (danielgent) [reply src/routes/tsoa/users.ts:135]: Should this just be the same as the `@Put` below? :thinking: 

:smile:

[the remark this answers is not in the document]
Comment (markusahlstrand) [reply src/routes/tsoa/users.ts:119]: This looks correct. The react-admin always uses the PUT endpoint so this has more or less never been used.

**Review by markusahlstrand — APPROVED** [review]
LGTM!

**Comment by danielgent** [comment]
No no @markusahlstrand , I assigned it to you to see if you knew why :smile: 


I can try and investigate more though when I'm next working. _maybe_ tomorrow or later tonight

Currently it's always creating a new user, but maybe i can debug the model code :thinking:

**Comment by danielgent** [comment]
I've confused myself with this PR :smile: 

I won't try and fix any _existing_ user endpoints

**Pull request closed without merging**